includes/class-api.php

```diff
@@ -153,6 +153,20 @@ public function get_addons() {
 			];
 		}
 
+		// Catch responses that don't contain an array of add-ons.
+		if ( ! isset( $response['addons'] ) ) {
+			return [
+				'addons' => [],
```

Comment:
Need to specify a "Requires PHP" line in the readme.

Reply:
Added #21.